src/ert/config/summary_config.py

```diff
@@ -32,6 +32,9 @@ def __post_init__(self) -> None:
 
     def read_from_file(self, run_path: str, iens: int) -> xr.Dataset:
         filename = self.input_file.replace("<IENS>", str(iens))
+        if "<GEO_ID>" in self.input_file and "geo_realization_" in run_path:
```

Comment:
Can we avoid putting this in the summary config? Would like to avoid details of everest leaking into here.

Reply:
Sure I can try to do that